mod-inn-reach/src/main/resources/volaris/mod-inn-reach/features/marc-transformation-options-settings.feature

```diff
@@ -14,23 +14,104 @@ Feature: MARC transformation options settings
 
     * configure headers = headersUser
 
-  @Undefined
-  Scenario: Get all MARC transformation options settings
-    * print 'Get all MARC transformation options settings'
+    * callonce variables
 
-  @Undefined
-  Scenario: Get MARC transformation options settings by id
-    * print 'Get MARC transformation options settings by id'
+    * print 'Prepare central servers'
+    * callonce read(featuresPath + 'central-server.feature@create')
+    * def centralServer1 = response.centralServers[0]
+    * def centralServer2 = response.centralServers[1]
+    * def notExistedCentralServerId1 = globalCentralServerId1
 
-  @Undefined
+  @create
   Scenario: Create MARC transformation options settings
+    * configure headers = headersUser
```

Comment:
headers are already set in background section

Reply:
Deleted everywhere